airbyte-integrations/connectors/source-twilio/source_twilio/spec.json

```diff
@@ -10,7 +10,7 @@
       "account_sid": {
         "title": "Account ID",
         "description": "Twilio account SID",
-        "airbyte_secret": true,
+        "airbyte_secret": false,
```

Comment:
If there is a spec change, it is a breaking change. We need to bump the version to 1.0.0 and add a `twilio-migrations.md` file.
But first, do we really need to change this?

Reply:
@btkcodedev I don't think this change is critical. Account ID doesn't sound like a secret variable and just thought it doesn't need to be a secret. I'll revert it to not cause a breaking change.